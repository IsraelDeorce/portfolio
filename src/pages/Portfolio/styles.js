import styled from 'styled-components'

export const Portfolio = styled.div`
  display: flex;
  flex-direction: column;
  min-width: 100%;
  > * {
    display: flex;
    flex-direction: column;
  }
  > :nth-child(even) {
    background-color: ${({ theme }) => theme.c_menu};
  }
  > :nth-child(odd) {
    background-color: ${({ theme }) => theme.c_background};
  }
`
