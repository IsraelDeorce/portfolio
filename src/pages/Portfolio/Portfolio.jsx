import Home from './Home/Home'
import Skills from './Skills/Skills'
import Experience from './Experience/Experience'
import * as S from './styles'

function Portfolio() {
  return (
    <S.Portfolio>
      <Home/>
      <Skills/>
      <Experience/>
    </S.Portfolio>
  )
}

export default Portfolio
