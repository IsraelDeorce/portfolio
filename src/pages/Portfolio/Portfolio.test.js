import { HashRouter as Router } from 'react-router-dom'
import Portfolio from './Portfolio'

describe('<Portfolio/>', () => {
  describe('Snapshot', () => {
    it('renders without crashing using shallow', () => {
      expect(shallow(<Portfolio />)).toMatchSnapshot()
    })

    it('renders without crashing using render', () => {
      expect(render(<Router><Portfolio /></Router>)).toMatchSnapshot()
    })

    it('renders without crashing using mount', () => {
      expect(mount(<Router><Portfolio /></Router>)).toMatchSnapshot()
    })
  })
})
