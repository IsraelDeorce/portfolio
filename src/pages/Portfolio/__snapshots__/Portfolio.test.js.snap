// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Portfolio/> Snapshot renders without crashing using shallow 1`] = `
<styled.div>
  <Home />
  <About />
  <Skills />
  <Experience />
  <Contact />
</styled.div>
`;
